Connected web3 provider at 10.195.3.55:8016
Date.now(): 1579678635915 startTime: 1579678672045 delay: 36133
Deploying smart contract from 0xdC55B8A1d3A0D44A62CD788a82bEC1C10485ab41
TX HASH:
 0xe59f07a47aec0d7f038659d1b38ff6a0af65a0c2bf8c3b38141c7457581c617a
RECEIPT:
 {
  blockHash: '0x55d7a708c503cd2c4295d0addaae6716ffe3912225542292c62a88d37ef101b3',
  blockNumber: 171,
  contractAddress: '0x159D15f0fBE11ec0b69C96Fb76C8368DA2E909Ad',
  cumulativeGasUsed: 218495,
  from: '0xdc55b8a1d3a0d44a62cd788a82bec1c10485ab41',
  gasUsed: 218495,
  logs: [],
  logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
  status: true,
  to: null,
  transactionHash: '0xe59f07a47aec0d7f038659d1b38ff6a0af65a0c2bf8c3b38141c7457581c617a',
  transactionIndex: 0
}
Contract at: 0x159D15f0fBE11ec0b69C96Fb76C8368DA2E909Ad

Evaluating  7000txs...

Sending TXS started at 1579678672100 ...
startingBlock: 208
... Sending TXS finished at 1579678773163
Total send duration: 101063
... Waiting for mining process to finish! ...
(node:10707) UnhandledPromiseRejectionWarning: Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at listOnTimeout (internal/timers.js:501:9)
    at processTimers (internal/timers.js:475:7)
(node:10707) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:10707) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
ERROR:
 Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at listOnTimeout (internal/timers.js:501:9)
    at processTimers (internal/timers.js:475:7)
========================================================
========================================================

/home/ubuntu/benchprezz/benchmark/tpsUnsigned.js:113
                throw 'Blockchain capabilities EXCEEDED - ABORTING test run!';
                ^
Blockchain capabilities EXCEEDED - ABORTING test run!
